lib/colors.js

```diff
@@ -31,3 +31,12 @@ function wrap(color, text, resetColor) {
     return text;
 }
 exports.wrap = wrap;
+
+/**
+ * Return a string with IRC colors stripped away.
+ * @param {String} line The IRC message.
+ * @return {String} An IRC message stripped of colors.
+ */
+exports.stripColors = function stripColors(line) {
+    return line.replace(/[\x02\x1f\x16\x0f]|\x03\d{0,2}(?:,\d{0,2})?/g, '');
```

Comment:
Looks like https://www.npmjs.com/package/irc-colors does everything irc colors. Probably worthwhile switching to that, but out of scope for this PR. I'll open an issue.